scipy/sparse/linalg/tests/test_special_sparse_arrays.py

```diff
@@ -153,7 +158,166 @@ def test_dot(self, grid_shape, bc, dtype):
             y = lap.dot(x.astype(dtype))
             assert x.shape == y.shape
             assert y.dtype == dtype
+            if x.ndim == 2:
+                yy = lap.toarray() @ x.astype(dtype)
+                assert yy.dtype == dtype
+                np.array_equal(y, yy)
 
     def test_boundary_conditions_value_error(self):
         with pytest.raises(ValueError, match="Unknown value 'robin'"):
             LaplacianNd(grid_shape=(6, ), boundary_conditions='robin')
+
+            
+class TestSakurai:
+    """
+    Sakurai tests
+    """
+    tested_types = [np.int32, np.int64] + REAL_DTYPES + COMPLEX_DTYPES
```

Comment:
Please add the same comment (as below) to the `tested_types` here as well.

Reply:
no longer used